src/components/LanguageButtons.js

```diff
@@ -12,14 +12,15 @@ function LanguageButtons({ page }) {
   const history = useHistory();
   const { t, i18n } = useTranslation();
 
+  //Changes the page's language as well as the language of the pathname
   const changeLanguage = (lng) => {
     history.push(t(`pathnames.${page}`));
```

Comment:
Cool to see how you combined string interpolation and the i18n translation function to update the pathname